src/scripts/components/navBar.js

```diff
@@ -1,32 +1,33 @@
 import renderToDom from '../helpers/renderToDom';
+import logo from '../../../instructions/hhpw-record.png';
 
 const navBar = () => {
   const domString = `
-    <nav id="navBarDiv" class="navbar navbar-expand-lg bg-light">
+  <nav id="navBarDiv" class="navbar navbar-expand-lg navbar-dark">
     <div class="container-fluid">
-      <span class="icon" style="span" width:50px></span>
-      <a class="navbar-brand" href="#">
-      </a>
       <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
       </button>
       <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <ul class="navbar-nav">
+          <li id="logo-image">
+            <img id="navLogo" src=${logo}
```

Comment:
looks like the img closing tag is missing? also, quotes around src=""

Reply:
I don't believe the img tag needs to close in this case and since the src is an interpolated variable it doesn't need the quotes, I believe. It renders correctly, which makes me think that's probably accurate.

Reply:
Yes, img is self-closing, meaning it doesn't need this ("</img>") but i meant the ">" at the end. I'm fine with leaving the quotes off though. My concern is that, even though it's rendering correctly now, it could throw off stuff later.